external_search.py

```diff
@@ -1103,7 +1103,6 @@ def __init__(self):
             'sort_author_keyword' : ['sort_author'],
             'integer': ['series_position', 'work_id'],
             'long': ['last_update_time'],
-            'float': ['random'],
```

Comment:
I'm not rebuilding the index from scratch, so migrated ES6 indexes will have a float field called 'random' that doesn't get used, but that's fine -- it's optional.